src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -138,13 +150,12 @@ export const getModuleDetailsUma = async (
     UMA_ORACLE_ABI,
     provider
   );
-
   const latestBlock = await provider.getBlock('latest');
   // modify this per chain. this should be updated with constants for all chains. start block is og deploy block.
   // this needs to be optimized to reduce loading time, currently takes a long time to parse 3k blocks at a time.
-  const oGstartBlock = network === '1' ? 17167414 : 0;
-  const oOStartBlock = network === '1' ? 16636058 : 0;
-  const maxRange = network === '1' ? 3000 : 10000;
+  const oGstartBlock = getDeployBlock(network, 'OptimisticGovernor');
+  const oOStartBlock = getDeployBlock(network, 'OptimisticOracleV3');
+  const maxRange = network === '1' || network === '5' ? 3000 : 10000;
```

Comment:
include goerli here to limited block ranges. not sure if strictly required, but cant hurt